logic.js

```diff
@@ -0,0 +1,142 @@
+let currentPage = 1;
```

Comment:
should be class base implementation, do not directly create variable in any file, it will be accessible globally. which is bad practice.

Reply:
Done